src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests_ListPatterns.cs

```diff
@@ -21,760 +21,107 @@ public PatternParsingTests_ListPatterns(ITestOutputHelper output) : base(output)
-                N(SyntaxKind.RecursivePattern);
-                {
-                    N(SyntaxKind.LengthPatternClause);
-                    {
-                        N(SyntaxKind.OpenBracketToken);
-                        N(SyntaxKind.RecursivePattern);
-                        {
-                            N(SyntaxKind.PredefinedType);
-                            {
-                                N(SyntaxKind.StringKeyword);
-                            }
-                            N(SyntaxKind.LengthPatternClause);
-                            {
-                                N(SyntaxKind.OpenBracketToken);
-                                N(SyntaxKind.RelationalPattern);
-                                {
-                                    N(SyntaxKind.GreaterThanToken);
-                                    N(SyntaxKind.NumericLiteralExpression);
-                                    {
-                                        N(SyntaxKind.NumericLiteralToken, "0");
-                                    }
-                                }
-                                N(SyntaxKind.CloseBracketToken);
-                            }
-                        }
-                        N(SyntaxKind.CloseBracketToken);
-                    }
-                }
-            }
-            EOF();
-        }
-
-        [Fact]
-        public void LengthPattern_14()
-        {
-            UsingExpression(@"c is [[_]]");
-
-            N(SyntaxKind.IsPatternExpression);
-            {
-                N(SyntaxKind.IdentifierName);
-                {
-                    N(SyntaxKind.IdentifierToken, "c");
-                }
-                N(SyntaxKind.IsKeyword);
-                N(SyntaxKind.RecursivePattern);
-                {
-                    N(SyntaxKind.LengthPatternClause);
-                    {
-                        N(SyntaxKind.OpenBracketToken);
-                        N(SyntaxKind.RecursivePattern);
-                        {
-                            N(SyntaxKind.LengthPatternClause);
-                            {
-                                N(SyntaxKind.OpenBracketToken);
-                                N(SyntaxKind.DiscardPattern);
-                                {
-                                    N(SyntaxKind.UnderscoreToken);
-                                }
-                                N(SyntaxKind.CloseBracketToken);
-                            }
-                        }
-                        N(SyntaxKind.CloseBracketToken);
-                    }
-                }
-            }
-            EOF();
-        }
-
-        [Fact]
-        public void LengthPattern_15()
+        public void ListPattern_01()
         {
-            UsingExpression(@"c is []",
-                // (1,7): error CS8504: Pattern missing
-                // c is []
-                Diagnostic(ErrorCode.ERR_MissingPattern, "]").WithLocation(1, 7));
+            UsingExpression(@"c is [[],] v");
```

Comment:
Do we still have a test for `is [[]]` (without comma)?
The comments below seem out-of-date.

Reply:
yes `SlicePattern_01`

Reply:
`SlicePattern_01` below is about `[..]`. I was asking about `[[]]`.